src/luarocks/deps.lua

```diff
@@ -1,147 +1,13 @@
 
---- Dependency handling functions.
--- Dependencies are represented in LuaRocks through strings with
--- a package name followed by a comma-separated list of constraints.
--- Each constraint consists of an operator and a version number.
--- In this string format, version numbers are represented as
--- naturally as possible, like they are used by upstream projects
--- (e.g. "2.0beta3"). Internally, LuaRocks converts them to a purely
--- numeric representation, allowing comparison following some
--- "common sense" heuristics. The precise specification of the
--- comparison criteria is the source code of this module, but the
--- test/test_deps.lua file included with LuaRocks provides some
--- insights on what these criteria are.
+--- High-level dependency related functions.
 local deps = {}
-setmetatable(deps, { __index = require("luarocks.core.deps") })
 
 local cfg = require("luarocks.core.cfg")
-local manif = require("luarocks.core.manif")
+local manif = require("luarocks.manif")
```

Comment:
please do it in this PR so we document in the history why the change was made :)